app/timetables/tests/test_models.py

```diff
@@ -235,3 +237,62 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class EventTest(TestCase):
+    """Tests the Event model."""
+
+    def setUp(self):
+        self.future_date = datetime.strptime('07 08 2016 05 30', '%d %m %Y %H %M')
+        self.earlier_date = datetime.strptime('06 07 2016 05 30', '%d %m %Y %H %M')
+        self.timetable = Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=self.earlier_date,
+            date_modified=self.future_date
+        )
+
+    def test_event_create(self):
+        Event.objects.create(
+            name='some event',
+            timetable=self.timetable,
+            start_date=self.earlier_date,
+            end_date=self.future_date,
+        )
+        evt = Event.objects.get(timetable=self.timetable)
+        self.assertEqual(evt.timetable.id, self.timetable.id)
+
+    def test_event_end_time_less_than_start_time_cannot_be_saved(self):
+        evt = Event(
+            name='Special',
+            timetable=self.timetable,
+            start_date=self.future_date,
+            end_date=self.earlier_date
+        )
+
+        self.assertRaises(ValidationError, evt.save)
+
+    def test_event_end_time_same_with_start_time_cannot_be_saved(self):
+        event = Event(
+            name='Special',
+            timetable=self.timetable,
+            start_date=self.earlier_date,
+            end_date=self.earlier_date
+        )
+
+        self.assertRaises(ValidationError, event.save)
+
+    def test_event_uniqueness(self):
+        event_data = {
```

Comment:
With the event setup in `setUp` we won't need to repeat this and line `296` would be enough to test